jekyll/_cci2/browser-testing.md

```diff
@@ -54,197 +54,28 @@ For more information about working with Headless Chrome,
-{% raw %}
-```
-# This code is licensed from CircleCI to the user under the MIT license. See
-# https://circleci.com/orbs/registry/licensing for details.
+```yaml
 version: 2.1
-description: >
-  Encapsulates interactions with Lambdatest tunnel
 
-commands:
-  install:
-    steps:
-      - run: 
-          name: "Downloading tunnel binary"
-          command: |
-            wget http://downloads.lambdatest.com/tunnel/linux/64bit/LT_Linux.zip
-      
-      - run: 
-          name: "Extracting tunnel binary"
-          description: "Installing dependencies - unzip, wget and ucommon-utils"
-          command: |
-            sudo apt-get update && sudo apt-get install -y wget unzip ucommon-utils
-            if [ "$(md5sum LT_Linux.zip | awk '{print $1}')" == "0279d9a29b2346e90ba237fc5b12d254" ]; then  echo "match";  else echo "Lambda tunnel binary has corrupted. Please rerun this workflow."; exit 1; fi 
-            unzip LT_Linux.zip
-    
-  open_tunnel:
-    parameters:
-      tunnel_name:
-        description: This feature will help you use the available tunnels from the selected head account which is the main account for performing parallel tunnel testing. While using this feature you are required to select both tunnel name and head tunnel.
-        type: string
-        default: ""
-    steps:
-      - run: 
-          name: "Executing tunnel binary"
-          background: true
-          command: |
-            : ${LAMBDATEST_EMAIL:?"Required Env Variable - LAMBDATEST_EMAIL not found!"}
-            : ${LAMBDATEST_USERNAME:?"Required Env Variable - LAMBDATEST_USERNAME not found!"}
-            : ${LAMBDATEST_KEY:?"Required Env Variable - LAMBDATEST_KEY not found!"}
-            ./LT -user ${LAMBDATEST_EMAIL} -key ${LAMBDATEST_KEY} <<# parameters.tunnel_name >> -tunnelName <<parameters.tunnel_name>> <</ parameters.tunnel_name>>
-            #wait for Lambda tunnel to be up
-            sleep 40
-  
-  close_tunnel:
-    steps:
-      - run:
-          name: Close Lambdatest Tunnel
-          command: killall LT
+orbs:
+  lambda-tunnel: lambdatest/lambda-tunnel@0.0.1
    
 jobs:
-  with_tunnel:
-    description: Use Lambdatest Tunnel
-    parameters:
-      docker:
-        type: string
-        default: "circleci/node:8.9.4"
-      tunnel_name:
-        description: Name your lambda tunnel and make sure to use this tunnel_name desired capability in your test
-        type: string
-        default: ""
-      steps:
-        type: steps
-        description: Steps to execute once the Lambdatest Tunnel is available
-    docker: 
-      - image: <<parameters.docker>>
+  lambdatest/with_tunnel
```

Comment:
```suggestion
  lambdatest/with_tunnel:
```
I think this might be what's causing the failing build.

Reply:
Oh well, worth a try, thanks Tyler! :-)